templates/App/_partials/_nav/_shared.html.twig

```diff
@@ -29,7 +31,10 @@
                 </li>
             {% endfor %}
         </ul>
-    </div>
+</li>
+
+<li class="separate-item sidebar-element">
+    <div></div>
```

Comment:
What is this for? Just to get match of some CSS?

Reply:
Yes, we need it to create a stylised line to separate language selection from the rest of items.
With `<div></div>` it looks like this:
![image](https://github.com/museum-engineering-project/koillectionPG/assets/86925036/d6dff6c0-62a5-405d-85ef-9ec93860e941)
And without `<div></div>` like this:
![image](https://github.com/museum-engineering-project/koillectionPG/assets/86925036/378a6f5e-5704-449c-922f-f5e42a35cc02)